pom.xml

```diff
@@ -90,22 +97,24 @@
         <dependency>
             <groupId>biz.paluch.logging</groupId>
             <artifactId>logstash-gelf</artifactId>
-            <version>1.12.0</version>
+            <version>1.14.0</version>
         </dependency>
+
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <scope>test</scope>
```

Comment:
While they are implicitly set, I think it makes sense to keep the dependency scope listed here to show that they are indeed only for tests.

Reply:
Sure, I will amend commit.